rgsync/redis_gears_write_behind.py

```diff
@@ -173,14 +200,14 @@ def func(r):
                     if kInHash.startswith('_'):
                         continue
                     if kInHash not in keys:
-                        msg = 'Could not find %s in hash %s' % (kInHash, r['key'])
+                        msg = 'AddToStream: Could not find %s in hash %s' % (kInHash, r['key'])
                         WriteBehindLog(msg)
                         raise Exception(msg)
                     data.append([kInDB, value[kInHash]])
         execute('xadd', self.GetStreamName(self.connector.TableName()), '*', *sum(data, []))
```

Comment:
I think we are doing tostring to the data here, if so then why parsing it with json.loads from the first place?

Reply:
I didn't change this code - just added a comment. Legitimately, no idea...

Reply:
I know you did not change it, I am saying that the code is doing a tostring, so no need to do the json.loads you are doing ..